src/Chapter1.hs

```diff
@@ -632,7 +638,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = secondToLastDigit + lastDigit n
+  where
+    tens = fst (divMod (abs n) 10)
```

Comment:
Cool usage of divs and mods! Also great work on reusing the `lastDigit` function.

Here, I see that you only use the div part of the `divMod` result. So instead of `fst` and `divMod` you can simply use `div` function 🙂